siatest/renter/renter_test.go

```diff
@@ -75,3 +79,49 @@ func testUploadDownload(t *testing.T, tg *siatest.TestGroup) {
 		t.Error(err)
 	}
 }
+
+// testDownloadMultipleLargeSectors downloads multiple large files (>5 Sectors)
+// in parallel and makes sure that the downloads are blocking each other.
+func testDownloadMultipleLargeSectors(t *testing.T, tg *siatest.TestGroup) {
```

Comment:
It looks like this test assumes a ratelimit. What it should probably do is fetch the current ratelimits, check that they are sufficiently low for this test to make sense, and then `t.Error` if not.